api/src/main/java/org/openmrs/module/rwandaemr/integration/UpdateClientRegistryTask.java

```diff
@@ -0,0 +1,50 @@
+package org.openmrs.module.rwandaemr.integration;
+
+import org.apache.commons.lang.time.StopWatch;
+import org.openmrs.Location;
+import org.openmrs.LocationTag;
+import org.openmrs.Visit;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.VisitService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.emrapi.EmrApiConstants;
+import org.openmrs.module.emrapi.adt.AdtService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Custom implementation to close stale visits
+ * This mimics the AdtService#CloseStaleVisitsTask, but is executed by Spring and adds additional logging
```

Comment:
nit, looks like this comment needs to be updated

Reply:
Thanks